src/system_window.cpp

```diff
@@ -256,8 +256,10 @@ SystemWindow::SystemWindow( Settings& settings )
 	viewport_size_.Width ()= static_cast<unsigned int>( width  / scale );
 	viewport_size_.Height()= static_cast<unsigned int>( height / scale );
 
+	#ifndef __APPLE__
 	use_gl_context_for_software_renderer_= !is_opengl && settings_.GetOrSetBool( "r_software_use_gl_screen_update", true );
-	if( is_opengl )
+	#endif
+    if( is_opengl )
```

Comment:
Remove unnecessary whitespace changes in this line (and maybe in other lines).

Reply:
fixed